contributors/contributors/github-sponsorship.md

```diff
@@ -0,0 +1,11 @@
+# Mattermost GitHub Sponsorship
+At Mattermost, we would like to empower our key contributors and sponsor them for 1 year. This is not a financial reward, but mostly a recognition.
+With [GitHub Sponsors](https://docs.github.com/en/github/supporting-the-open-source-community-with-github-sponsors/about-github-sponsors#about-github-sponsors) allowing the developer community to financially support the people and organizations who design, build, and maintain the open source projects they depend on, directly on GitHub, we can now launch this new Community Program.
+## Who is Eligible?
+_Mattermost Github sponsorship_ is always on the table for [MVP](https://developers.mattermost.com/contribute/mvp/) (“Most Valued Professional”) type people in any of the [available regions](https://github.com/sponsors) supported by GitHub.
+## Process Outline
+1. A Mattermost staff member recommends someone for sponsorship and writes a short motivation to support their recommendation.
+2. [MLT](https://handbook.mattermost.com/company/about-mattermost/list-of-terms#mlt) exec member approves the recommendation.
+3. The request is sent to the Finance team.
+4. An email is sent to the potential sponsor letting them know about the next steps in the process.
```

Comment:
4. Jason Balis reach the potential sponsor letting them know about the next steps in the process.